File: Java Fundamentals/DataTypesAndVariables/src/SumDigits.java
import java.util.Scanner;

public class SumDigits {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        String numString = new String(scan.nextLine());
        int sum = 0;

        for (int d = 0; d < numString.length(); d++) {
            int currentDigit = Integer.parseInt(String.valueOf(numString.charAt(d)));
            sum += currentDigit;
        }

        System.out.println(sum);
    }
}

File: JS Front-End/JS Syntax Fundamentals/SumDigits.js
function sumDigits(num){
    let sumOfDigits = 0;
    let numString = Math.abs(num).toString();
    let digit;
    for(let d = 0; d < numString.length; d++){
        sumOfDigits += parseInt(numString.charAt(d));
    } 

    console.log(sumOfDigits);
}

sumDigits(245678);
sumDigits(97561);
sumDigits(543);